app/upw/templates/assessment-help.njk

```diff
@@ -0,0 +1,301 @@
+{% extends "common/templates/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "common/templates/components/widgets/macro.njk" import widgets %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block pageTitle %}
+{{ pageTitle }}
+{% endblock %}
+
+{% block header %}
+{%- set usingBrowserBack = true -%}
+{% include "common/templates/partials/header.njk" %}
+{% endblock %}
+
+{% block content %}
+{{ super() }}
+
+<div class="govuk-grid-row">
+        <h1 class="govuk-heading-xl">
+            {{ pageTitle }}
+        </h1>
+    <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-heading-l">
+            Task list
+        </h2>
+        <p>
+            The task list will update automatically as you
+            work through the assessment. <br>
```

Comment:
Rather than wrap the entire text it may be worth making each paragraph it's own `p` element to avoid using the `br`s

Reply:
Unless I am missing something here